rllib/examples/connectors/run_connector_policy.py

```diff
@@ -52,7 +52,7 @@ def run(checkpoint_path):
         print(f"step {step}", obs, action)
 
         # Step environment forward one more step.
-        obs, _, done, _ = env.step(action[0])
+        obs, _, done, _ = env.step(action)
```

Comment:
This is because we have removed the batch dim from local_policy_inference output? nice.

Reply:
Yep